.github/workflows/nix-update-flake-lock.yml

```diff
@@ -9,6 +9,7 @@ on:
 jobs:
   lockfile:
     runs-on: ubuntu-latest
+    if: github.repository == 'Alexays/Waybar'
```

Comment:
Maybe `if: github.event_name != 'schedule' || github.repository == 'Alexays/Waybar'`?

I found `workflow_dispatch` support useful when I had to update `flake.lock` without any knowledge about Nix.

Reply:
I was gonna do that, since we do that in nixvim to allow people to manually run it. Didn't know if people would find it helpful here, too.